Sources/TemplateKit/ViewRenderer.swift

```diff
@@ -0,0 +1,21 @@
+import Foundation
+
+/// View renderers are responsible for loading the files paths given and caching if needed.
+///
+/// View renderers are also responsible for accepting a Node for templated responses.
```

Comment:
oh man,`Node`. That is an old doc block.

Reply:
This should mention codable instead.

Reply:
Use the same as TemplateRenderer: `/// Renders raw template data (bytes) to `View`s.` ?

Reply:
maybe:

> `ViewRenderer` returns a `View` for a supplied path an `Encodable` object.

ViewRenderer doesn't have to work directly with raw bytes at all, for instance with mocked versions